test/mixins.test.js

```diff
@@ -79,11 +79,23 @@ describe('Model class', function() {
     properties.city.should.eql({ type: String, required: true });
   });
 
+  it('should fail to apply an undefined mixin class', function(done) {
```

Comment:
Since this test completes on the same turn of the event loop, there is no need to use `done` callback.
Instead of 

```js
it('...', function(done) {
  var memory = ...;
  // ...
  should...
  done();
});
```

write simply:

```js
it('...', function() {
  var memory = ...;
  // ...
  should...
});
```